Containers/mastercontainer/cron.sh

```diff
@@ -47,8 +47,14 @@ while true; do
     # Remove sessions older than 24h
     find "/mnt/docker-aio-config/session/" -mindepth 1 -mmin +1440 -delete
 
+    # Remove nextcloud-aio-domaincheck container
+    sudo -u www-data docker container remove nextcloud-aio-domaincheck --force
```

Comment:
we should only remove the container if it is stopped. You should be able to get it with e.g.  docker ps --filter "status=exited"

Reply:
```suggestion
    if [ "$(docker ps --format "{{.Names}}" --filter "status=exited" | grep -w "nextcloud-aio-domaincheck" | head -n 1)" == "nextcloud-aio-domaincheck" ]; then
        sudo -u www-data docker container remove nextcloud-aio-domaincheck
    fi
```

Reply:
```suggestion
    if [ "$(docker ps --format "{{.Names}}" --filter "status=exited" | grep -q "^nextcloud-aio-domaincheck$")" == "nextcloud-aio-domaincheck" ]; then
        sudo -u www-data docker container remove nextcloud-aio-domaincheck
    fi
```

Reply:
```suggestion
    if docker ps --format "{{.Names}}" --filter "status=exited" | grep -qw "^ttt$"; then
        sudo -u www-data docker container remove nextcloud-aio-domaincheck
    fi
```